src/js/components/user/connect.tsx

```diff
@@ -79,7 +79,8 @@ const ConnectModal = ({
 
   const handleConnect = () => {
     window.location.assign(
-      addUrlParams(window.api_urls.salesforce_production_login(), {
+      addUrlParams(window.api_urls.salesforce_login(), {
+        custom_domain: 'login',
```

Comment:
@davisagli Why is it preferable to set this to a magic string, rather than relying on the logic above (https://github.com/oddbird/MetaShare/pull/226/files#diff-0cc7a69625eda623a982686a93a0d96dR37-R40) to do the same thing?

Reply:
@jgerigmeyer which logic? I don't think that link is taking me to the right place

Reply:
I mean this logic in `SalesforceOAuth2Adapter`:

```py
        if custom_domain == "login" or not custom_domain:
            base_url = "https://login.salesforce.com"
        else:
            base_url = "https://{}.my.salesforce.com".format(custom_domain)
```

I'm curious the value of using `"login"` instead of relying on the `not custom_domain` branch.

Reply:
@jgerigmeyer oh, got it. I started only using the `not custom_domain` check, but then realized there was an edge case if the user starts logging in with a custom domain but doesn't finish, then comes back and tries to log in via the production login page. Then we still get the prior custom_domain from the session. So using the magic string is to make sure that we always override that.

Reply:
Ah, ok. I'm alright with this in that case.